schema-multiversion-launch-week.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Schema multi-version early access preview goes live today
```

Comment:
I don't know if we should call it an early access preview as it rolls into the public beta for our serverless postgres offering. What do you think about just announcing this functionality and then talking about it as part of the general public beta or postgres?

Reply:
Sure. For some reason I thought we were calling this an EAP. Happy to remove it.

Reply:
Removed